src/i18n/cmd/ru.json

```
@@ -0,0 +1,203 @@
+{
+  "lang": "en-US",
```

This should be `ru` :)
```suggestion
  "lang": "ru",
```

---

Oops :) Fixed.